lib/modules/manager/tflint-plugin/plugins.ts

```diff
@@ -0,0 +1,84 @@
+import is from '@sindresorhus/is';
+import { logger } from '../../../logger';
+import { regEx } from '../../../util/regex';
+import { GithubReleasesDatasource } from '../../datasource/github-releases';
+import type { PackageDependency } from '../types';
+import type { ExtractionResult, TFLintManagerData } from './types';
+import { keyValueExtractionRegex } from './util';
+
+export function extractTFLintPlugin(
+  startingLine: number,
+  lines: string[],
+  pluginName: string
+): ExtractionResult {
+  let lineNumber = startingLine;
+  const deps: PackageDependency<TFLintManagerData>[] = [];
+  const dep: PackageDependency<TFLintManagerData> = {
+    managerData: {
+      pluginName: pluginName,
+    },
+  };
+
+  let braceCounter = 0;
+  do {
+    // istanbul ignore if
+    if (lineNumber > lines.length - 1) {
+      logger.debug(`Malformed TFLint configuration file detected.`);
+    }
+
+    const line = lines[lineNumber];
+
+    // istanbul ignore else
+    if (is.string(line)) {
+      // `{` will be counted with +1 and `}` with -1.
+      // Therefore if we reach braceCounter == 0.
+      // We have found the end of the tflint configuration block
```

Comment:
```suggestion
      // Therefore if we reach braceCounter == 0 then we found the end of the tflint configuration block.
```

This should be one sentence.